bin/lib/zopen-generate

```diff
@@ -17,6 +17,8 @@ echo "What is the project name?"
 name=$(getInput)
 echo "Provided a description of the project:"
 description=$(getInput)
+echo "Provide a url link to the license contents: (e.g. https://raw.githubusercontent.com/mirror/make/master/COPYING). (if none, press enter)"
```

Comment:
Need to require an apache license here

Reply:
Add a data file and valid license options to choose from